.github/workflows/lint.yml

```diff
@@ -8,6 +8,7 @@ on:
 
 jobs:
   quick-checks:
+    name: Lint / quick-checks
```

Comment:
Aw man I didn't realize it would make the signal show up like
<img width="377" alt="image" src="https://user-images.githubusercontent.com/31798555/158259820-482f299a-35c4-44b7-bba5-09a6744527f3.png">